mobile/library/common/network/connectivity_manager.cc

```diff
@@ -357,7 +354,7 @@ ConnectivityManagerImpl::getAlternateInterfaceSocketOptions(NetworkType network)
 envoy_netconf_t
 ConnectivityManagerImpl::addUpstreamSocketOptions(Socket::OptionsSharedPtr options) {
   envoy_netconf_t configuration_key;
-  NetworkType network;
+  int network;
```

Comment:
`int` certainly makes sense here for a bitmap, but I wonder if it would be better to encapsulate this functionality in a class, where it has methods like `contains(NetworkType)`, `add(NetworkType)`, `remove(NetworkType)` .. WDYT?

If you'd like to get this in sooner though, that could be done in a follow up

Reply:
I think it'll be quite messy to encapsulate this in a class, because we'd need to pass that through jni.